randomize/plugin.py

```diff
@@ -19,8 +19,37 @@
 from nose.util import isclass, isgenerator, transplant_func, transplant_class
 import random
 import unittest
+from random import getrandbits
+from nose.tools import nottest
 
 log = logging.getLogger(__name__)
+CLASS_SPECIFIC_RANDOMIZE_TESTS_FIELD_NAME = 'randomize_tests_seed'
+
+
+@nottest
```

Comment:
Yep. Nose detects the decorator as a test method whenever I use it.
Other remarks addressed.